kpatch-build/kpatch-build

```diff
@@ -697,81 +697,81 @@ for i in $FILES; do
 	# In RHEL 7 based kernels, copy_user_64.o misuses the .fixup section,
 	# which confuses create-diff-object.  It's fine to skip it, it's an
 	# assembly file anyway.
-	[[ $DISTRO = rhel ]] || [[ $DISTRO = centos ]] || [[ $DISTRO = ol ]] && \
-		[[ $i = arch/x86/lib/copy_user_64.o ]] && continue
+	[[ "$DISTRO" = rhel ]] || [[ "$DISTRO" = centos ]] || [[ "$DISTRO" = ol ]] && \
+		[[ "$i" = arch/x86/lib/copy_user_64.o ]] && continue
 
-	[[ $i = usr/initramfs_data.o ]] && continue
+	[[ "$i" = usr/initramfs_data.o ]] && continue
 
-	mkdir -p "output/$(dirname $i)"
+	mkdir -p "output/$(dirname "$i")"
 	cd "$SRCDIR"
-	find_kobj $i
-	if [[ $KOBJFILE = vmlinux ]]; then
-		KOBJFILE=$VMLINUX
+	find_kobj "$i"
+	if [[ "$KOBJFILE" = vmlinux ]]; then
+		KOBJFILE="$VMLINUX"
 	else
 		KOBJFILE="$TEMPDIR/module/$KOBJFILE"
 	fi
-	cd $TEMPDIR
+	cd "$TEMPDIR"
 	if [[ -e "orig/$i" ]]; then
 		# create-diff-object orig.o patched.o kernel-object output.o Module.symvers patch-mod-name
 		"$TOOLSDIR"/create-diff-object "orig/$i" "patched/$i" "$KOBJFILE" \
 			"output/$i" "$SRCDIR/Module.symvers" "${MODNAME//-/_}" 2>&1 |tee -a "$LOGFILE"
 		check_pipe_status create-diff-object
 		# create-diff-object returns 3 if no functional change is found
-		[[ $rc -eq 0 ]] || [[ $rc -eq 3 ]] || ERROR=$(expr $ERROR "+" 1)
-		if [[ $rc -eq 0 ]]; then
-			[[ -n $ERROR_IF_DIFF ]] && die $ERROR_IF_DIFF
+		[[ "$rc" -eq 0 ]] || [[ "$rc" -eq 3 ]] || ERROR="$(expr $ERROR "+" 1)"
+		if [[ "$rc" -eq 0 ]]; then
+			[[ -n "$ERROR_IF_DIFF" ]] && die "$ERROR_IF_DIFF"
```

Comment:
`$rc`, `$ERROR`, `$ERROR_IF_DIFF` don't need quoting, here and below

Reply:
Dito.